api/client/commands.go

```diff
@@ -1829,10 +1829,11 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 
 func (cli *DockerCli) CmdLogs(args ...string) error {
 	var (
-		cmd    = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
-		follow = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
-		times  = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
-		tail   = cmd.String([]string{"-tail"}, "all", "Output the specified number of lines at the end of logs (defaults to all logs)")
+		cmd      = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
+		follow   = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
+		times    = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
+		tail     = cmd.String([]string{"-tail"}, "all", "Output the specified number of lines at the end of logs (defaults to all logs)")
+		truncate = cmd.Bool([]string{"-truncate"}, false, "Fetch and truncate logs of a container")
```

Comment:
I think what is is trying to say is that `docker logs` already fetches the logs so saying that in the flag is redundant.  The flag should just say truncates the containers logs after they are sent.